### DISCUSSION TITLE
Fetch data from Supabase with NodeJS (Error: invalid input syntax for type timestamp)

### DISCUSSION THREAD

**Discussion by AlexanderCryptoOwl** [supabase/supabase#5468] [Questions] [answered]
using Supabase JS library to fetch data from my tables. My code works fine if I use directly filter .eq but not with .or for comparing timestamps. I tried different combos such as defined toISOString outside of the function but to no prevail. 

Documentation: https://supabase.com/docs/reference/javascript/or

Error: 

    {
      message: 'invalid input syntax for type timestamp: "ts.toISOString("',
      code: '22007',
      details: null,
      hint: null

Code:

    var ts = new Date();
    ts.setHours(ts.getHours() - 168)
    
    const supabaseNotifications = async() => {
        let { data, error } = await supabase
        .from('table3')
        .select('*, table1!inner(coin, coin_abr, combo_change, combo_signal, combo_prev_signal), table2!inner(email, created_at, is_subscribed)')
        .eq('table1.combo_change', true)
        .or('created_at.gt.ts.toISOString(),is_subscribed.eq.true', { foreignTable: "table2" })
    
        if(error) {
            console.error(error)
            return
        }
        console.log(data)
    }
    
    supabaseNotifications()

**Comment by steve-chavez** [comment, marked as answer]
Try it like:
```js
.or(`created_at.gt.${ts.toISOString()},is_subscribed.eq.true`, { foreignTable: "table2" })
```

**Reply by AlexanderCryptoOwl** [reply]
Thank you so much that did the trick!
